### ISSUE TITLE
Cannot start charon after do the setup explained in the manual

### ISSUE THREAD

**Issue by moellenbeck** [process-engine/charon#75]
#### Describe your issue

Cannot start charon after install it explained in https://github.com/process-engine/documentation/blob/develop/introduction/installation.md#installation

#### How can others reproduce the issue?

```
➜  charon git:(develop) npm start

> charon@1.0.0-rc1 start /Users/moellenbeck/projects/process-engine/reps/charon
> http-server -p 9000

Starting up http-server, serving ./
Available on:
  http://127.0.0.1:9000
  http://192.168.178.38:9000
Hit CTRL-C to stop the server
[Fri Dec 08 2017 21:52:21 GMT+0100 (CET)] "GET /" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/604.4.7 (KHTML, like Gecko) Version/11.0.2 Safari/604.4.7"
[Fri Dec 08 2017 21:52:21 GMT+0100 (CET)] "GET /scripts/vendor-bundle.js" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/604.4.7 (KHTML, like Gecko) Version/11.0.2 Safari/604.4.7"
[Fri Dec 08 2017 21:52:21 GMT+0100 (CET)] "GET /scripts/vendor-bundle.js" Error (404): "Not found"
[Fri Dec 08 2017 21:52:21 GMT+0100 (CET)] "GET /favicon.ico" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/604.4.7 (KHTML, like Gecko) Version/11.0.2 Safari/604.4.7"
^Chttp-server stopped.

➜  charon git:(develop) npm -v
4.2.0
➜  charon git:(develop) node -v
v7.10.1
➜  charon git:(develop)
```
#### What's your setup

- OS (`Windows/OS X/Linux` + `version`): ProductName:	Mac OS X
ProductVersion:	10.13.2
BuildVersion:	17C85a
- Node (`node --version`): v7.10.1
- NPM (`npm --version`): 4.2.0
- Docker (`docker version --format '{{.Server.Version}}'`): 17.09.0-ce

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by heikomat** [comment]
If you compare the readme of charon and the manual in the documentation you'll notice, that the build-step is missing in the manual. The readme for Charon says:
- `npm install`
- `npm run build`
- `npm start`

While the manual only says:
- `npm install`
- `npm start`

**Comment by moellenbeck** [comment]
Did't work cause I have no aurelia-cli (au) installed and I won't that. So we need to update the readme of this repo and link to the manual (want to have only one documentation). And we need a solution to start charon without having au-cli installed.

**Comment by heikomat** [comment]
@moellenbeck you don't need the aurelia-cli installed globally. It is listed in charons dev-devpendencies.

When `npm install`ing, the aurelia-cli gets installed into charons node_modules-folder. When running `npm run build` it runs `au build --prod`, but because it is run as package.json-script, this uses the locally installed aurelia-cli, the on ein charons node_modules-folder

**Comment by moellenbeck** [comment]
Oh, so I can't `NODE_ENV=production npm install` ? That's what I have done.

**Comment by heikomat** [comment]
your command is equivalent to `npm install --production`. If you do that, only `dependencies` get installed, but no `devDependencies`. Because the aurelia-cli is in the dev-dependencies, building charon won't like that.

i'd really like to be able to differentiate between `dependencies`, `devDependencies` and something like `buildDependencies`, but sadly the latter one doesn't exist.

**Comment by moellenbeck** [comment]
> your command is equivalent to npm install --production.
@heikomat I know - but the fact is, that we need a installation without devDependencies, cause I did't want to change charon. I only want to use it.

**Comment by heikomat** [comment]
@moellenbeck this means moving nearly all devDependencies (except for testing-things and browser-sync) to the regular dependencies. If you say this should be done, than that would be the solution.

**Comment by moellenbeck** [comment]
@heikomat no what I mean is, that we need publish charon to npm with all stuff build and only install charon via `npm install @processengine/charon` or so.

**Comment by heikomat** [comment]
ok, that would also work. because charon will the be built already, and because it then is a static website, the only thing we'd need in the regular dependencies would be `http-server`. everything else could go to the devDependencies

**Comment by moellenbeck** [comment]
and make it possible to provide the`baseRoute` as maybe processEngineBaseURL to the startup script.

**Comment by heikomat** [comment]
this could be difficult, but doable. We usually configure things pre-build, in the [`environments`-files](https://github.com/process-engine/charon/tree/develop/aurelia_project/environments). we'd need to find a way to load the config from somewhere else after the build.

i'm pretty sure someone at 5minds has done this before, but i'd suggest that it is out of the scope of this issue, and should be discussed/worked on in a separate issue.

**Comment by moellenbeck** [comment]
@heikomat and make charon look more professional ;-)

**Comment by heikomat** [comment]
This seems to be a timing or caching problem within the process-engine:

When i slow down my browser, so that charon is slower, this issue doesn't happen

**Comment by heikomat** [comment]
![ezgif-5-ffe671be32](https://user-images.githubusercontent.com/20770029/33876312-7894b04a-df25-11e7-86d8-c21049145c6b.gif)

**Comment by heikomat** [comment]
Closed in favour of #86
